pkg/occlient/occlient.go

```diff
@@ -2204,7 +2219,10 @@ func (c *Client) CopyFile(localPath string, targetPodName string, targetPath str
 	}()
 
 	// cmdArr will run inside container
-	cmdArr := []string{"tar", "xf", "-", "-C", targetPath, "--strip", "1"}
+	cmdArr := []string{"tar", "xf", "-", "-C", targetPath}
```

Comment:
Why do we execute an external command instead of using the tar library that go provides? https://golang.org/pkg/archive/tar/

Reply:
This command get's executed inside the supervisord container. It's basically needed because we send a stream of bytes to the pod that are "tarred", so inside the "pod" we need to untar them.

Also, I didn't add this code, I just altered the code that was already there to work for our use case :P

Reply:
So, we're assuming that container has tar installed? I'm not sure that that's the case in every s2i container (and eventually non s2i).

Maybe we should open an issue to investigate if that can happen.

Reply:
@jorgemoralespou That is indeed the case. The code seems to be initially inspired by `kubectl`.

See: https://github.com/kubernetes/kubernetes/blob/040cc281f4849dee05d35e4f2fa2ee36729dea22/pkg/kubectl/cmd/cp/cp.go#L45

Reply:
@kadel Can you shine some light here please?